tests/test_publish.py

```diff
@@ -219,3 +219,48 @@ def test_publish_pattern(self, standalone_tsm_context, published_data_reader):
         assert len(published_data) == len(expected_results)
         for published_data_point, expected_result in zip(published_data, expected_results):
             assert published_data_point.boolean_value == expected_result
+
+
+@pytest.mark.pin_map("publish.pinmap")
+class TestPerSiteMultiSite:
+    def test_publish_per_site_float_1d(self, standalone_tsm_context, published_data_reader):
+        test_data = [1150.0, 1952.5, -4.0]
+        standalone_tsm_context.publish_per_site(test_data)
+        published_data = published_data_reader.get_and_clear_published_data()
+        for published_data_point in published_data:
+            site_index = standalone_tsm_context.site_numbers.index(published_data_point.site_number)
+            assert published_data_point.double_value == test_data[site_index]
```

Comment:
Let's update all of these tests to test non-empty pin and published data id.
```suggestion
            assert published_data_point.double_value == test_data[site_index]
            assert published_data_point.published_data_id == "id"
            assert published_data_point.pin == "pin"
```

Reply:
https://github.com/ni/nitsm-python/pull/144/commits/323d157548ad1e446c27998a864c7da22b4a7a38